File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/test/java/simulacionCesta/factories/ProductFactoryTest.java
package simulacionCesta.factories;

import org.junit.jupiter.api.Test;
import simulacionCesta.models.Product;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ProductFactoryTest {

    @Test
    public void prodcutFactoryCreateTest() {
        Product product = ProductFactory.create();
        // Comprobar que la cesta final contiene una cesta no vacía
        assertNotNull(product);
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/test/kotlin/simulacionCesta/factories/ProductFactoryTest.kt
package simulacionCesta.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class ProductFactoryTest {
    @Test
    fun prodcutFactoryCreateTest() {
        val product = ProductFactory.create()
        // Comprobar que la cesta final contiene una cesta no vacía
        Assertions.assertNotNull(product)
    }
}